cellengine/utils/api_client/APIClient.py

```diff
@@ -119,6 +130,56 @@ def update_entity(self, experiment_id, _id, entity_type, body) -> dict:
             json=body,
         )
 
+    def _get_path(self, entity: CE) -> str:
+        fullpath = f"{self.base_url}/{entity.path}"
+        return fullpath
+
+    def create(self, entity: CE, **kwargs) -> CE:
+        """Create a local entity on CellEngine."""
+        if type(entity) is list:
+            return self.create_multiple(entity, **kwargs)  # type: ignore
+        path = self._get_path(entity)
+        structure_class = entity.__class__
+        if structure_class.__bases__[0] is not object:
+            structure_class = structure_class.__bases__[0]
+        body = converter.unstructure(entity)
+
+        if body["_id"] == "None" or body["_id"] is None:
+            del body["_id"]  # https://github.com/primitybio/cellengine/issues/5800
+
+        res = self._post(path, json=body, params=kwargs)
+        return converter.structure(res, structure_class)  # type: ignore
+
+    # TODO: add type annotation for entities
+    def create_multiple(self, entities: List[CE], **kwargs) -> List[CE]:
```

Comment:
I'm not sure how useful this will be. We should be using the bulk APIs where possible. It looks just about as easy for the user to call `create` in a loop. Not sure how much efficiency is gained here.

Reply:
It's a little confusing here, but this actually does use the bulk API for gate creation. The function exists to strip the `_id` from gates where it equals "None". I'll take another look at this and see if I can improve it.

Reply:
Ah sorry, bad read on my part.

Reply:
Worse writing on my part, and in fairness I did have a `create_multiple` at some point in the past that posted gates individually. This basically comes down to a workaround around #108

Reply:
Okay, I've reimplemented this as a single function, and added overloading to boot :pirate_flag:

Reply:
My other PRs will need a quick rebase after this